requirements.txt

```diff
@@ -1,3 +1,13 @@
+black==24.2.0
+click==8.1.7
 Faker==23.3.0
+mypy-extensions==1.0.0
+packaging==24.0
+pathspec==0.12.1
+platformdirs==4.2.0
+prompt-toolkit==3.0.43
 python-dateutil==2.9.0.post0
 six==1.16.0
+tomli==2.0.1
+typing_extensions==4.10.0
+wcwidth==0.2.13
```

Comment:
where do all these reqs come from?

I created new venv with only these 2 and it works:
```
Faker==23.3.0
prompt_toolkit==3.0.43
```

Reply:
I use `pipreqs` lib to generate requirements.txt. It mostly does a good job

Reply:
thanks for the feedback, let me try to fix those deps

Reply:
Q: Do you know if pipreqs needs to be in deps too?

Reply:
The person trying our app will not need it, so my guess is no. Plus it didn't put itself in reqs